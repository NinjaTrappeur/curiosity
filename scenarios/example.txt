user get USER-1 --short
user signup alice secret alice@example.com
user get USER-1 --short
