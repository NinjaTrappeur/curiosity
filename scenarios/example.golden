1: user get USER-1 --short
No such user.
2: user signup alice secret alice@example.com
User created: USER-1
3: user get USER-1 --short
USER-1 alice
